setup.cfg

```diff
@@ -3,7 +3,7 @@ name = up-python
 author = Neelam Kushwah
 author_email = neelam.kushwah@gm.com
 description = UProtocol Python SDK
-version = 1.5.2
+version = 1.5.8
```

Comment:
version needs to be 0.1.0 and we cannot use 1.5 per not I sent in the slack channel

Reply:
This file is not in use and has been deleted in PR #17. Once PR #17 is merged, I will rebase it.